### PR TITLE
KeyBindings, Refactored Sort Documents,  added instructions to Tutorial

### PR DESCRIPTION
**Pull request by eddypjr**
### Types of Changes

- [x]  Refactored the Sort Documents button in the User Dashboard to be in line with the other drop down buttons in the application 
- [x]  Added a dark mode button in the User Dashboard
- [x]  Added keybindings for Undo, Redo, Export, Save, Save Project As, Delete, Sign in
- [x] Works for both Mac and Windows
- [x] Refactored the Undo and Redo logic to function independently from other Components
- [x] Added Keybinding instructions to the Tutorial section
- [x] useEffect and useCallback for keybindings added to ComponentPanel.tsx, DeleteProjects.tsx, ExportButton.tsx, OpenProjects.tsx, SignIn.tsx, SaveProjectButton.tsx, RightContainer.tsx

### Purpose

- Keybindings improve the user experience
- To have Undo and Redo work independently from other Components

### Tutorial
**Mac**
-             Export Project: Command + e
-             Undo: Command + z
-             Redo: Command + Shift + z
-             Save Project As: Command + s
-             Save Project: Command + shift + s
-             Delete HTML Tag on Canvas: Backspace
-             Delete Project: Command + Backspace
-             Open Project: Command + o
**Windows**
-             Export Project: Control + e
-             Undo: Control + z
-             Redo: Control + Shift + z
-             Save Project As: Control + s
-             Save Project: Control + shift + s
-             Delete HTML Tag on Canvas: Backspace
-             Delete Project: Control + Backspace
-             Open Project: Control + o

### Screenshots
**Tutorial for Keybindings in Help, Tutorial, Canvas**
Screen Shot 2021-03-17 at 10 41 19 AM

**Screenshot for changes in RightContainer.tsx, Canvas,tsx (in tutorial), and componentReducer.ts**
Screen Shot 2021-03-17 at 10 35 15 AM

**Screenshot of User Dashboard with refactored Sort Documents button and Dark/Light Mode**
Screen Shot 2021-03-17 at 10 47 47 AM

### CODE REVIEW

**Review by slhoehn — APPROVED** [review]
good job!

**Comment by elenaconn** [comment]
Looks good!

**Review by elenaconn — APPROVED** [review]
Looks good!

File: server/graphQL/schema/typeDefs.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const Project = gql`
     userId: ID!
     username: String!
     createdAt: String
+    comments: String
```
Comment (khuongdn16) [inline server/graphQL/schema/typeDefs.js:26]: Will resolve in the next PR

**Review by khuongdn16 — APPROVED** [review]
New code for implementing server logic for Comments will be in a separate PR.

**Review by Anthonytorrero — APPROVED** [review]
Approved

**Review by Anthonytorrero — COMMENTED** [review]
Approved

**Pull request merged**